### DISCUSSION TITLE
platform/actors/development/actor-definition/input-schema/specification/v1

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#1357] [Comments]
# platform/actors/development/actor-definition/input-schema/specification/v1

Learn how to define and validate a schema for your Actor's input with code examples. Provide an autogenerated input UI for your Actor's users.

https://docs.apify.com/platform/actors/development/actor-definition/input-schema/specification/v1

**Comment by ryanleecode** [comment]
Can we get a JSON Schema please

**Reply by jancurn** [reply]
Do you mean JSON schema for the input schema itself? It's here: https://github.com/apify/apify-shared-js/blob/master/packages/input_schema/src/schema.json
